README.rst

```diff
@@ -99,20 +109,16 @@ Documentation
 The documentation is hosted online at `api.mongodb.org <http://api.mongodb.org/lua/current/>`_.
 
 You will need ldoc_ installed to generate the
-documentation. Documentation can be generated by running the following command in the source directory:
-
-.. code-block:: bash
+documentation. Documentation can be generated by running the following command in the source directory.::
```

Comment:
Do try to avoid fixing more than one issue in a single commit. In this case it seems you're both adding CMake support, and updating some of the documentation content, and updating the docs' formatting markup.